assets/transformify.yaml

```diff
@@ -0,0 +1,23 @@
+---
+
+- type: application
+  name: Automate
```

Comment:
for asset names that are very generic (e.g. Cohere's Command model, Transformify's Automate platform), we should include the organization name. I.e. rename to `Transformify Automate`.